readme/dev/accessibility.md

```diff
@@ -0,0 +1,91 @@
+	- After installation, TalkBack can be enabled as on a physical Android device, from the settings app.
+- **Web**:
+	- Enable the operating system's screen reader (see "Desktop: Setting up a screen reader").
+
+On a physical mobile device, VoiceOver and TalkBack share certain gestures:
+- **To focus the next item**: Swipe from left to right.
+- **To focus the previous item**: Swipe from right to left.
+- **To jump focus to a particular part of the screen**: Tap once on the part of the screen to focus.
+- **To click on the focused item**: Double-tap anywhere on the screen.
+
+## Adding automated tests to prevent regressions
+
+The desktop and mobile apps use different frameworks for checking for accessibility-related regressions: The desktop app uses [Playwright](https://playwright.dev/) and the mobile app uses [react-native-testing-library](https://callstack.github.io/react-native-testing-library/).
+
+### Desktop: Automated accessibility testing
+
+Existing Playwright automated tests can be found in `packages/app-desktop/integration-tests`. These tests can be run with `yarn test-ui` from `packages/app-desktop`. It's possible to run a subset of all tests using `yarn test-ui -g "test name search pattern"`, where `-g` stands for "grep".
+
+For general accessibility issues (e.g. contrast, missing labels), we use [`@axe-core/playwright`](https://www.npmjs.com/package/@axe-core/playwright). This library scans everything visible on the screen and returns a set of errors. See `integration-tests/wcag.spec.ts` for existing tests that make use of this library.
+
+Keyboard interface tests should go in the other test files. For example, `richText.spec.ts` currently includes several tests related to using the editor with a keyboard. While writing these tests, the [expect.toBeFocused](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-focused), [keyboard.press](https://playwright.dev/docs/api/class-keyboard#keyboard-press), and [getByRole](https://playwright.dev/docs/api/class-frame#frame-get-by-role) APIs might be particularly useful.
+
+The [getByRole](https://playwright.dev/docs/api/class-frame#frame-get-by-role) API can also be used to verify that components have specific [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) information.
+
+### Mobile: Automated accessibility testing
+
+On mobile, [react-native-testing-library](https://callstack.github.io/react-native-testing-library/) also provides a [getByRole](https://callstack.github.io/react-native-testing-library/docs/api/queries#by-role) selector that can be used to ensure that accessibility information is present. See, for example, [this test for list of installed plugins](https://github.com/laurent22/joplin/blob/bf58a52394947acc42f8ca527b3ce22464d989c3/packages/app-mobile/components/screens/ConfigScreen/plugins/PluginStates.installed.test.tsx#L231).
```

Comment:
Likewise we should link in priority to our own documentation (assuming this is documented somewhere)

Reply:
This doesn't seem to be documented anywhere.